### ISSUE TITLE
How to list the active fixes?

### ISSUE THREAD

**Issue by luzfcb** [hhatto/autopep8#667]
Hello, first, thanks for the hard work on this project. 

I'm trying to figure out what exactly are the fixes that autopep8 applies by `default`, and what are the fixes that autopep8 will use when for example, a project has multiple configuration files with sections that define `select` and `ignore`.

for example, running `autopep8 --list-fixes` shows that there are options W291, W292, W293, W391

now let's assume I have two config files:

.flake8:
```
[flake8]
select = W29,W39
```

setup.cfg
```
[pycodestyle]
ignore = W292
```

let's assume that there is an optional argument called `--active-only`

```bash
autopep8 --list-fixes --active-only
```
The output would be something like:
```
W291 - Remove trailing whitespace.
W293 - Remove trailing whitespace.
W391 - Remove trailing blank lines.
```

if I also include `--aggressive` and `--experimental` like:

```bash
autopep8 --list-fixes --active-only -aaa --experimental
```

The output will show W291, W29, W391 and the other fixes that autopep8 apply when aggressive and experimental is used.

In other words, is there any way to get the list of active fixes after autopep8 load all configuration files?
